a1 : [0.0, -0.3, 0.4]
tile_size : 0.0375
#Normd vec for x and y (no z as chess pieces normally slide of the table when slanted...)
orientation: [-1, -1]